### DISCUSSION TITLE
Article Counter

### DISCUSSION THREAD

**Discussion by hoanghai1212** [OrchardCMS/OrchardCore#9913] [Q&A]
Hi Orchard Core Team, I'm using Orchard Core CMS for my project for now and everything great. Now I'm going to make an component call Popular Story but I can't find out any solution to know which my article is popular. I have an idea if I can get amount of people who click to article so I can know which is popular. Does we have any feature support for this counter for now or any plan to do this in the feature ? 
By the way I'm using Orchard Core CMS as Headless CMS
Thank you team.

**Comment by Skrypt** [comment]
Google Tag Manager module. Can be used to collect statictics on a specific button click event which gets reported in Google Analytics.

**Comment by hoanghai1212** [comment]
> Google Tag Manager module. Can be used to collect statictics on a specific button click event which gets reported in Google Analytics.

Tks for your solution, I'll take a look on this 👍
